.github/workflows/benchmark.yml

```diff
@@ -4,15 +4,19 @@ on:
   # Submit your review with a comment body containing "#benchmark"
   pull_request_review:
     types: [ submitted ]
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   build_modified_ckb:
     name: Build Modified CKB
-    runs-on: [ self-hosted, Linux, X64, building ]
+    runs-on: ubuntu-latest
```

Comment:
I change the runner from self-hosted to github-hosted. The reason for this is that "Benchmark" and "Sync-Mainnet" are low-frequency tests. We can run multiple tests at the same time using github-hosted runners.